plugin/al/translators/tests/testTranslators.py

```diff
@@ -561,7 +561,9 @@ def testDirectionalLight_PluginIsFunctional(self):
         mc.AL_usdmaya_ProxyShapeImport(file='./testDirectionalLight.usda')
         self.assertTrue(Tf.Type.Unknown != Tf.Type.FindByName('AL::usdmaya::fileio::translators::DirectionalLight'))
         self.assertEqual(len(mc.ls('directionalLightShape1')), 1)
-        self.assertEqual(len(mc.ls(type='directionalLight')), 1)
+        # Latest (2022-05-24) Maya preview release creates proxy light nodes in the scene for USD lights, 
+        # so the query below returns 2 now. Comment out the check. 
+        #self.assertEqual(len(mc.ls(type='directionalLight')), 1)
```

Comment:
I think we should keep the check there and assert that the value is 2. That way if the behavior changes we can detect it. We'll have to add some python code to detect the Maya version and change the test based on that.

Something along the lines of :

`mayaUtils.mayaMajorVersion() > 2023`

should be useful.